cavern/src/main/java/org/opencadc/cavern/CavernInitAction.java

```diff
@@ -138,6 +142,25 @@ private void initNodePersistence() {
             log.error("Failed to create JNDI Key " + jndiNodePersistence, ex);
         }
     }
+
+    private void initIdentityManager() {
+        final String configuredIdentityManagerClassName = System.getProperty(IdentityManager.class.getName());
+        if (configuredIdentityManagerClassName == null) {
+            throw new InvalidConfigException("No existing IdentityManager found.  Ensure that the " + IdentityManager.class.getName()
+                                                 + " System Property is set to a valid implementation.");
+        }
+
+        // Assuming there isn't more than one Cavern deployed within a JVM.
+        PosixIdentityManager.JNDI_NODE_PERSISTENCE_PROPERTY = this.jndiNodePersistence;
+
+        // To be used by the PosixIdentityManager to wrap the existing IdentityManager.
+        System.setProperty(PosixIdentityManager.WRAPPED_IDENTITY_MANAGER_CLASS_PROPERTY, configuredIdentityManagerClassName);
+
+        // Override the existing IdentityManager.
+        System.setProperty(IdentityManager.class.getName(), PosixIdentityManager.class.getName());
+
+        log.debug("IdentityManager set to: " + PosixIdentityManager.class.getName() + " from " + configuredIdentityManagerClassName);
```

Comment:
This I would log at info level and make it clear that 
```
log.info(IdentityManager.class.getName() + " = " + PosixIdentityManager.class.getName() 
     + " delegating to " + configuredIdentityManagerClassName);
```
This code only runs once and it's good to always log the results of init from config.